idaes/apps/grid_integration/coordinator.py

```diff
@@ -353,6 +356,64 @@ def _clone_tracking_model(self):
 
         return
 
+    def _update_static_params(self, gen_dict):
+
+        """
+        Update static parameters in the Prescient generator parameter data dictionary.
+
+        Args:
+            gen_dict: Prescient generator parameter data dictionary.
+
+        Returns:
+            None
+        """
+
+        for param, value in self.bidder.bidding_model_object.model_data:
+            if param == "gen_name" or value is None:
+                continue
+            elif param == "p_cost":
+                curve_value = convert_marginal_costs_to_actual_costs(value)
+                gen_dict[param] = {
+                    "data_type": "cost_curve",
+                    "cost_curve_type": "piecewise",
+                    "values": curve_value,
+                }
+                if "p_fuel" in gen_dict:
+                    gen_dict.pop("p_fuel")
+            else:
+                gen_dict[param] = value
+
+            if param == "startup_cost" and "startup_fuel" in gen_dict:
```

Comment:
Is there a reason this is outside the if, elif, else statement above? If so, just add a one-line comment explaining